File: api-examples/cat/count/js/attribute_search_spatial_circle_responseFilter.js
var settings = {
  "url": "https://pudx.catalogue.iudx.org.in/catalogue/v1/count?attribute-name=(tags)&attribute-value=((pollution))&lat=18.4&lon=73.9&radius=200000&attribute-filter=(tags,id)",
  "method": "GET",
  "timeout": 0,
};

$.ajax(settings).done(function (response) {
  console.log(response);
});

File: api-examples/cat/count/java/attribute_search_spatial_circle_responseFilter.java
OkHttpClient client = new OkHttpClient().newBuilder()
  .build();
Request request = new Request.Builder()
  .url("https://pudx.catalogue.iudx.org.in/catalogue/v1/count?attribute-name=(tags)&attribute-value=((pollution))&lat=18.4&lon=73.9&radius=200000&attribute-filter=(tags,id)")
  .method("GET", null)
  .build();
Response response = client.newCall(request).execute();
